### DISCUSSION TITLE
Using of PHP arrays triggers false positive alerts

### DISCUSSION THREAD

**Discussion by maxxer** [WeblateOrg/weblate#7914] [Q&A]
Hi.
I'd like to use our Weblate instance to translate a PHP extension, [yii2-usuario](https://github.com/2amigos/yii2-usuario), which uses php arrays as translation strings.

Beyond the source string display, which embeds *some code* which can be puzzling for non-technical translators, we are experiencing several false positives alerts like ending colons or dots: 

![screenshot-weblate -2022 07 22-12_19_30](https://user-images.githubusercontent.com/240201/180419703-afa26ea8-d564-4f7a-889f-83ab0af29e0f.png)

The extension is for Yii2, but the format is the same of Laravel. I tried both standard PHP and Laravel ones, but the result is the same. 

Did I miss a setting or something that will handle source strings correctly?

Thanks

Using Weblate 4.11.2 self hosted

**Comment by nijel** [comment]
Bilingual PHP files are not supported, see https://github.com/WeblateOrg/weblate/issues/2547
